RELEASES.md

```diff
@@ -0,0 +1,28 @@
+# Versioning and Release
+This document describes the versioning and release process of Harbor. This document is a living document, contents will be updated according to each releases.
+
+## Releases
+Harbor releases will be versioned using dotted triples, similar to [Semantic Version](http://semver.org/). For this specific document, we will refer to the respective components of this triple as `<major>.<minor>.<patch>`. The version number may have additional information, such as "-rc1,-rc2,-rc3" to mark release candidate builds for earlier access. Such releases will be considered as "pre-releases".
+
+### Major and Minor Releases
+Major and minor releases of Harbor will be branched from master when the release reaches to `RC(release candidate)` state. The branch format should follow `release/<major>.<minor>.0`. For example, once the release `v1.0.0` reaches to RC, a branch will be created with the format `release/1.0.0`. When the release reaches to `GA(General Available)` state, The tag with format `v<major>.<minor>.<patch>` and should be made with command `git tag -s v<major>.<minor>.<patch>`. The release cadency is around 3 months, might be adjusted based on open source event, but will communicate it clearly.
```

Comment:
The branch name looks like `release-1.0.0`

Reply:
thanks for the review. fixed it.